app/src/main/java/org/wildaid/ofish/ui/activity/ActivitiesFragment.kt

```diff
@@ -85,7 +85,7 @@ class ActivitiesFragment : BaseReportFragment(R.layout.fragment_activity) {
             }
 
             R.id.btn_next -> {
-                if (isFieldCheckPassed || isAllRequiredFieldsNotEmpty()) {
+                if (isFieldCheckPassed || validateForms()) {
```

Comment:
The method is changed. Do we need to check both conditions?

Reply:
This is just renaming of method, since it has side effect under the hood. 
Boolean value indicates the user has already passed validation. It prevents preventing user to navigate futher